api-key-thing-python/validator.py

```
@@ -0,0 +1,22 @@
+import sys, json, os, hashlib
+
+p = os.path.join(os.path.dirname(__file__), 'db.json')
+args = sys.argv[1:]
+
+if not args:
+    sys.exit('Usage: python validate_key.py <key>')
+
+def get_db():
+    try:
+        with open(p) as f: return json.load(f)
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)